### PR TITLE
Add Georgia Institute of Technology

### PR DESCRIPTION
**Pull request by kylekoza**

### CODE REVIEW

**Review by mxxcon — APPROVED** [review]

File: _data/education.yml [hunk 1]
```diff
@@ -115,6 +115,16 @@ websites:
       img: edx.png
       tfa: No
 
+    - name: Georgia Institute of Technology
+      url: https://www.gatech.edu
+      img: gatech.png
+      tfa: Yes
+      sms: No
```
Comment (stephengroat) [inline _data/education.yml]: if `sms` is not enabled, then please remove the tag

**Review by stephengroat — CHANGES_REQUESTED** [review]
thanks for the pr @kylekoza! if you could address the one issue, that'd be great!

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Please use this image instead.
gatech

File: _data/education.yml (hunk 1 above)
Comment (kylekoza) [reply _data/education.yml]: @stephengroat: I have removed the sms tag

**Comment by kylekoza** [comment]
@Carlgo11: I have changed out the image as suggested

**Review by stephengroat — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**